crates/pop-parachains/src/up/chain_specs.rs

```diff
@@ -183,7 +183,7 @@ mod tests {
 	#[tokio::test]
 	async fn chain_spec_generator_returns_none_when_no_match() -> anyhow::Result<()> {
 		let temp_dir = tempdir()?;
-		assert_eq!(chain_spec_generator("rococo-local", None, temp_dir.path()).await?, None);
```

Comment:
In this case leave the `rococo-local`, is a test that specifically tests that `chain_spec_generator` returns `None`  if the value is not one of the Runtimes we support